troposphere/static/js/components/common/image_request/ImageLicensing.jsx

```diff
@@ -17,7 +17,7 @@ export default React.createClass({
         });
     },
     removeLicenseFromList: function(license_id) {
-        console.log("Remove", license_id);
+        // console.log("Remove", license_id);
```

Comment:
Interesting fact - `<ImageLicensing />` does not appear to be used anywhere within Troposphere.

And ... the Image Request interaction appears to allow for removing licenses, using the common "tag-like" view: http://recordit.co/FU1k2haLjj